src/python/bow/bow.py

```diff
@@ -0,0 +1,224 @@
+import numpy as np
+import cv2
+import argparse
+from pathlib import Path
+import matplotlib.pyplot as plt
+
+from sklearn import preprocessing
+from sklearn.cluster import KMeans
+from sklearn.neighbors import KDTree
+
+kDefaultWidth = 640  # px
+kDefaultClusterSize = 400
+
+
+def listImagesInFolder(folderPath):
+    trainImageFiles = list(folderPath.glob("*.jpg"))
+    trainImageFiles.extend(list(folderPath.glob("*.png")))
+    return trainImageFiles
+
+
+def rescaleImageIfNeeded(image):
```

Comment:
nit: very minor, but you seem to have docs for the other functions and not for these. Maybe add docstring to these too?